bin/controller.py

```diff
@@ -52,7 +52,10 @@ def post_new():
     except ValueError as exc:
         raise bt.HTTPError(400, str(exc))
 
-    snippet = models.Snippet.create(code, max(maxusage, 0), lifetime)
+    try:
+        snippet = models.Snippet.create(code, max(maxusage, 0), lifetime)
+    except RuntimeError as exc:
+        raise bt.HTTPError(500, str(exc))
```

Comment:
```suggestion
        raise bt.HTTPError(508, str(exc))
```
What do you think about [`508`](https://developer.mozilla.org/fr/docs/Web/HTTP/Status/508) ?

Reply:
> peut être donné dans le contexte du protocol Web Distributed Authoring and Versioning